Source/WebKit/Scripts/preprocess-sandbox-profile-rule.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh -e
+
+eval ARCHS=(${ARCHS})
```

Comment:
It turns ARCHS into an array, so that later we can grab the first architecture with `${ARCHS[0]}`. This is analogous to what the Makefiles does, `$(word 1, $(ARCHS))`. And it's needed because we have to run the clang preprocessor against _some_ target triple, and Copy Files phases are executed once per build, not once per architecture.